man/timesyncd.conf.xml

```diff
@@ -91,6 +91,12 @@
         <varname>PollIntervalMaxSec=</varname> defaults to 2048 seconds.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ContactServersRetryIntervalSec</varname></term>
+        <listitem><para>Specifies the delaying attempts to contact servers after network is online. Takes a time value (in seconds).
+        Defaults to 30 seconds and must not be smaller than 5 seconds.</para></listitem>
```

Comment:
why the 5s limit? that sounds unnecessary? 1s should work too?

Reply:
I tested it 10/15 times. Something below 5 it's gets into a loop sometimes and gets too bz. system does not boot and hangs.